agasc/tests/test_multi_agasc_file.py

```diff
@@ -0,0 +1,40 @@
+import os
+import tables
+from ska_path import ska_path
+from astropy.table import Table
+import tempfile
+
+from .. import agasc
+
+DATA_ROOT = ska_path('data', 'agasc')
+tables_open_file = getattr(tables, 'open_file', None) or tables.openFile
+
+def test_multi_agasc():
+
+    tempdir = tempfile.mkdtemp()
+
+    # Make two custom agasc files from the miniagasc, using 20 stars from
```

Comment:
The second agasc file is confirming that you can switch back and forth between custom agasc files and it doesn't break anything and that whatever you are doing for the caching (either persistence or explicit caching) is appropriately set.